IssueTracker/IssueTracker.FileSystem/Repository/BlobData.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System.Runtime.CompilerServices;
+using Azure.Storage.Blobs;
+using Azure.Storage.Sas;
+
+[assembly: InternalsVisibleTo("IssueTracker.UnitTest")]
+namespace IssueTracker.FileSystem;
+public class BlobData : IBlobProvider
+{
+    private readonly IBlobConfigurationFactory _config;
+    private BlobServiceClient BlobServiceClient { get; set; }
```

Comment:
Consistency !!! Why is this a property and _config is not?

Reply:
I changed